foreman/data_refinery_foreman/surveyor/test_external_source.py

```diff
@@ -25,9 +25,19 @@ def test_queue_downloader_jobs(self, mock_send_task):
 
         sample_object_1 = Sample()
         sample_object_1.accession_code = "Sample1"
+        sample_object_1.platform_accession_code = "Illumina Genome Analyzer"
+        sample_object_1.platform_accession_name = "Illumina Genome Analyzer"
+        sample_object_1.technology = "RNA-SEQ"
+        sample_object_1.manufacturer = "ILLUMINA"
+        sample_object_1.source_database = "SRA"
         sample_object_1.save()
         sample_object_2 = Sample()
         sample_object_2.accession_code = "Sample2"
+        sample_object_2.platform_accession_code = "Illumina Genome Analyzer"
+        sample_object_2.platform_accession_name = "Illumina Genome Analyzer"
```

Comment:
Duplicated duplicated line

Reply:
Not quite, one is setting the code the other is setting the name.

Reply:
"The bond's Name. James Name" 

Pleased to... what? 

"Bond Name's the james" 

Are you alright? 

"Bames Nond's having a stronk, call a Bondulance"